crates/pallets/starknet/src/utils.rs

```diff
@@ -11,6 +11,14 @@ impl From<Error> for sc_cli::Error {
     }
 }
 
+impl From<Error> for String {
```

Comment:
`Error` should rather implement `Display`

Reply:
It's a more rustic way

Reply:
I think I'm doing this in the wrong way. Even if i implement `Display`, the compiler still throws an error that I'm missing `String`

Reply:
The true fix will be to use anyhow rather than `String` as the error type

Reply:
The true fix would be to use anyhow instead of string as your error type

Reply:
Can we leave that refactor to another PR? (I'll open an issue for that if u agree)

Reply:
Sure

Reply:
Don't forget to open the issue